packages/app-desktop/gui/NoteEditor/utils/contextMenuUtils.ts

```diff
@@ -30,11 +30,17 @@ export interface ContextMenuItems {
 	[key: string]: ContextMenuItem;
 }
 
-export async function resourceInfo(options: ContextMenuOptions): Promise<any> {
+export interface ResourceInfo {
+	resource: any;
+	resourcePath: string;
+	filename: string;
+}
+
+export async function resourceInfo(options: ContextMenuOptions): Promise<ResourceInfo> {
```

Comment:
As this type is not used anywhere, could you remove it please? Then just make the function return nothing (but not void!) as in `export async function resourceInfo(options: ContextMenuOptions) {`) and type inference will do the job.

Reply:
> type inference will do the job.

I'm not sure I understood this part, are you suggesting to to remove the `interface ResourceInfo`? If so, how can we specify the type.

Reply:
Oh sorry I just noticed that the type gets automatically inferred by doing so, wasn't aware of it.